contracts/cw20-bonding/src/contract.rs

```diff
@@ -23,6 +23,7 @@ use cw0::{must_pay, nonpayable};
 const CONTRACT_NAME: &str = "crates.io:cw20-bonding";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+#[entry_point]
```

Comment:
As above, please gate all of these `entry_point`s in all contracts with library feature